samples/migrations/migrate_qdrant_vectorstore_to_alloydb.py

```diff
@@ -68,9 +68,14 @@ def get_data_batch(
 
         for doc in docs:
             if doc.payload and doc.vector:
+                # You might need to update this data translation logic according to your field names
+                # id is the unqiue identifier for the content
                 ids.append(str(doc.id))
+                # page_content is the content which was encoded
                 contents.append(doc.payload["page_content"])
+                # vector is the vector embedding of the content
```

Comment:
Why does this and id need to be updated? This is a object property from the API

Reply:
resolved offline